src/components/ImageGeneration/GeneratedImageActions.tsx

```diff
@@ -70,6 +72,8 @@ export function GeneratedImageActions({
         onChange={(e) => handleCheckboxClick(e.currentTarget.checked)}
         label={!selectedCount ? 'Select all' : `${selectedCount} selected`}
         labelPosition="left"
+        mr={8}
+        styles={{ label: { paddingLeft: 8, paddingRight: 8, whiteSpace: 'nowrap' } }}
```

Comment:
These changes can also be done using tailwind. Instead of `styles`, you can target `classNames`. `p-2` is 8px

Reply:
```suggestion
        classNames={{ label: 'px-2 whitespace-nowrap' }}
```